docs: input.sensitive and output.sensitive default value. (#102)
terramate-io/terramate-docs: cli/orchestration/outputs-sharing.md

- `backend` _(required, string)_ - The name of the sharing backend that this input refers to.
- `from_stack_id` _(required, string)_ - The stack ID where the outputs will be loaded.
- `value` _(required, expression)_ - The expression that computes this input at runtime.

> - `sensitive` _(optional, boolean)_ - If the value is sensitive or not. The default is `true`.

Reviewer: ```suggestion
- `sensitive` _(optional, boolean)_ - Marks the value as sensitive. Defaults to `true`.
```

- `backend` _(required, string)_ - The name of the sharing backend that this input refers to.
- `value` _(required, expression)_ - The expression that exports the stack's resource.

> - `sensitive` _(optional, boolean)_ - If the value is sensitive or not. The default is `true`.

Reviewer: ```suggestion
- `sensitive` _(optional, boolean)_ - Marks the value as sensitive. Defaults to `true`.
```
